raiden/raiden_service.py

```diff
@@ -145,7 +145,7 @@ def __init__(
             raise ValueError('invalid private_key')
 
         self.tokennetworkids_to_connectionmanagers = dict()
-        self.identifier_to_results = defaultdict(list)
+        self.identifier_to_results = dict()
```

Comment:
Can you add type anotation to it (maybe the variables around, if it's easy too), please?

Reply:
sure.